CONTRIBUTING.md

```diff
@@ -152,12 +136,12 @@ $ git rebase upstream/master
 
 ### Step 5: Test
 
-Bug fixes and features **should come with tests**. Add your tests in the
-`test/parallel/` directory. For guidance on how to write a test for the Node.js
-project, see this [guide](./doc/guides/writing-tests.md). Looking at other tests
-to see how they should be structured can also help.
+Bug fixes and features should come with tests. Read the
+[guide for writing tests in Node.js](./doc/guides/writing-tests.md). Looking at
+other tests to see how they should be structured can also help. Add your
+tests in the `test/parallel/` directory if you are unsure where to put them.
 
-To run the tests on Unix / macOS:
+To run the tests (including code linting) on Unix / macOS:
```

Comment:
I suggest move all the technical stuff except `make test` / `vcbuild test` to a new `HOWTO test.md`